idaes/apps/matopt/README.md

```diff
@@ -13,7 +13,7 @@ Thank you for your interest in MatOpt. We would love to hear your feedback! Plea
 
 If you are using MatOpt, please consider citing:
 
--   Hanselman, C.L., Yin, X., Miller, D.C. and Gounaris, C.E., 2020. MatOpt: A Python package for nanomaterials design using discrete optimization. *In preparation*.
+-   Hanselman, C.L., Yin, X., Miller, D.C. and Gounaris, C.E., 2021. MatOpt: A Python package for nanomaterials discete optimization *[Preprint available](http://gounaris.cheme.cmu.edu/drafts/Draft_MATOPT.pdf)*.
```

Comment:
Typo in "discrete" in publication title.

Reply:
Thx